.github/ISSUE_TEMPLATE/post-release-branch-creation.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Post Release Branch Creation Tasks
+about: Tasks to perform after the rc.0 is cut and the upcoming release branch is created
+title: Post Release Branch Creation Tasks for v1.x.y-rc.0
+labels: sig/release, area/release-eng
+---
+
+## As a follow up on: <!-- #<issue> typically the rc.0 cut issue (currently the branch is created by krel during that cut process) -->
+
+## Tasks
+
+<!--
+
+Follow the docs here: https://github.com/kubernetes/sig-release/blob/master/release-engineering/handbooks/post-release-branch-creation.md
+
+Help? Ring @release-managers on slack!
+
+-->
+
+- [ ] Create a thread in #release-management: <!-- Paste link to Slack thread -->
+- [ ] Coordinate the next alpha release cut - <!-- mention @release-managers nudging them to perform the propedeutic tasks for the next alpha (e.g. setting up the new OBS project) -->
+- [ ] (optional) Remove jobs for EOL versions, **only** if agreed upon with Release Managers
```

Comment:
We talked about getting rid of removing jobs because it complicates things a lot. I think we should get rid of it from here to make it easier for Branch Managers (and this is already quite complicated).